client/components/common/Chip.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
Implement `readonly` prop and override current v-chip  (don't introduce new component) or use a workaround in the comment.